lib/streamlit/runtime/state/widgets.py

```diff
@@ -37,53 +34,16 @@
     from streamlit.runtime.scriptrunner import ScriptRunContext
 
 
-ElementType: TypeAlias = str
-
-# NOTE: We use this table to start with a best-effort guess for the value_type
-# of each widget. Once we actually receive a proto for a widget from the
-# frontend, the guess is updated to be the correct type. Unfortunately, we're
-# not able to always rely on the proto as the type may be needed earlier.
-# Thankfully, in these cases (when value_type == "trigger_value"), the static
-# table here being slightly inaccurate should never pose a problem.
-ELEMENT_TYPE_TO_VALUE_TYPE: Final[Mapping[ElementType, ValueFieldName]] = (
-    MappingProxyType(
-        {
-            "button": "trigger_value",
-            "button_group": "int_array_value",
-            "camera_input": "file_uploader_state_value",
-            "checkbox": "bool_value",
-            "chat_input": "string_trigger_value",
-            "color_picker": "string_value",
-            "component_instance": "json_value",
-            "data_editor": "string_value",
-            "dataframe": "string_value",
-            "date_input": "string_array_value",
-            "download_button": "trigger_value",
-            "file_uploader": "file_uploader_state_value",
-            "multiselect": "int_array_value",
-            "number_input": "double_value",
-            "plotly_chart": "string_value",
-            "radio": "int_value",
-            "selectbox": "int_value",
-            "slider": "double_array_value",
-            "text_area": "string_value",
-            "text_input": "string_value",
-            "time_input": "string_value",
-            "vega_lite_chart": "string_value",
-        }
-    )
-)
-
-
 def register_widget(
-    element_type: ElementType,
     element_proto: WidgetProto,
     deserializer: WidgetDeserializer[T],
     serializer: WidgetSerializer[T],
     ctx: ScriptRunContext | None,
     on_change_handler: WidgetCallback | None = None,
     args: WidgetArgs | None = None,
     kwargs: WidgetKwargs | None = None,
+    *,
```

Comment:
nit: can we move the keyword thing here up above `deserializer`? looks like most uses only use element_proto as positional

Reply:
yep, I will change it 👍